src/main/json/r6OperatorCounters.json

```diff
@@ -1503,6 +1503,33 @@
                     "properties": {
                         "Description": "Dokkaebi can use her Logic Bomb to ring Caveria's phone. This will reveal Caveria's position when roaming even if Silent Step is active."
                     }
+                },
+                {
+                  "id": "143",
+                  "type": "Hard Counter",
+                  "startNode": "48".
```

Comment:
The git diff is flagging up that there is a full stop instead of a comma at the end of this line. I think this needs checked.

When you've made all your changes, try running the app using the instructions in the readme to test them out.